test/unit/setup.js

```diff
@@ -18,3 +18,4 @@ if (global.document) {
   }));
   document.execCommand = jest.fn();
 }
+global.VERDACCIO_API_URL = 'http://localhost/-/verdaccio/'
```

Comment:
Here `global.VERDACCIO_API_URL` I'd use something different as `global.TEST_VERDACCIO_API_URL` then when you search by `VERDACCIO_API_URL` we don't confuse with `window.VERDACCIO_API_URL`